test/config/wfe2.json

```diff
@@ -100,8 +101,6 @@
 			]
 		],
 		"staleTimeout": "5m",
-		"authorizationLifetimeDays": 30,
-		"pendingAuthorizationLifetimeDays": 7,
```

Comment:
It looks like we might still be using these two settings.

Reply:
This has now been resolved.